symfony/plugins/orangehrmLeavePlugin/test/fixtures/AssignLeaveAPITest.yaml

```diff
@@ -0,0 +1,107 @@
+Config:
```

Comment:
I can't see any `ReportTo` fixtures here, seems you are depend on some other test case fixtures
Refer; `symfony/plugins/orangehrmLeavePlugin/test/fixtures/MyLeaveRequestAPITest.yaml`  line no: 71